app/src/main/java/com/example/sharingang/DetailedItemFragment.kt

```diff
@@ -27,9 +32,20 @@ class DetailedItemFragment : Fragment() {
             DataBindingUtil.inflate(inflater, R.layout.fragment_detailed_item, container, false)
 
         binding.item = args.item
-
         binding.shareButton.setOnClickListener { shareItem() }
 
+        viewModel.setUser(args.item.userId)
+        viewModel.user.observe(viewLifecycleOwner, { user ->
+            binding.username = "Posted by " + user?.name
```

Comment:
```suggestion
            binding.username = "Posted by ${user?.name}"
```

I prefer using string interpolation, but that's kind of a personal preference I guess

Reply:
Oh I thought I actually did that because Android Studio complained, but I guess I somehow reverted it.